bcipy/tasks/rsvp/main_frame.py

```diff
@@ -244,15 +117,17 @@ class DecisionMaker:
         """
 
     def __init__(self,
-                 min_num_seq,
-                 max_num_seq,
-                 decision_threshold=0.8,
                  state='',
                  alphabet=list(string.ascii_uppercase) + ['<'] + [SPACE_CHAR],
                  is_txt_stim=True,
                  stimuli_timing=[1, .2],
                  seq_constants=None,
-                 criteria_evaluator=CriteriaEvaluator.default()):
+                 stopping_evaluator=CriteriaEvaluator.default(min_num_seq=2,
+                                                              max_num_seq=10,
+                                                              threshold=0.8),
+                 stimuli_agent=RandomStimuliAgent(
+                     alphabet=list(string.ascii_uppercase) +
+                              ['<'] + [SPACE_CHAR])):
```

Comment:
There is a potential bug here if someone provides a custom alphabet but does not include a different stimuli_agent. The stimuli_agent parameter should be given a default value of `None` and constructed in the __init__ method if it's not provided, using the alphabet parameter. Also, the default value for alphabet should use the helpers/task.py so we're not repeating code.

Reply:
I need your input there. I am not sure how I can bundle the input parameter file with the classmethod.

Reply:
I'm proposing using the method signature:
   `..., stimuli_agent=None):`

then in the init method:
   `self.stimuli_agent = stimuli_agent or RandomStimuliAgent(alphabet=self.alphabet)`

Reply:
The constructor is updated to `..., stimuli_agent=None):` 

And in the attributes, it is defined as:
`self.stimuli_agent = RandomStimuliAgent(alphabet=self.alphabet)`